### PR TITLE
German translation update

### PR DESCRIPTION
**Pull request by SIRSteiner**

### CODE REVIEW

File: FlymFork/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -1,8 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!--
+    <!--
+ Copyright (c) 2012-2015 Frederic Julian
  Copyright (c) 2010-2012 Stefan Handschuh
- Copyright (c) 2012-2014 Frederic Julian
-      modified 2014-2016 Matthias Dill
```
Comment (yanus171) [inline FlymFork/src/main/res/values-de/strings.xml]: Why do you want to remove this copyright?

File: FlymFork/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -23,24 +22,24 @@
  THE SOFTWARE.
 -->
 <resources>
-
     <!-- GENERAL -->
-    <string name="about_flym">Über app / Help / Contact developer </string>
-    <string name="welcome_title">Wilkommen in Flym. Willst du einen Feed hinzufügen?</string>
+    <string name="app_name">Flym Fork</string>
+    <string name="about_flym">Info - Hilfe - Support</string>
+    <string name="welcome_title">Willkommen bei Flym Fork. Möchten Sie einen Feed hinzufügen?</string>
```
Comment (yanus171) [inline FlymFork/src/main/res/values-de/strings.xml]: Change app name to Handy News Reader please

File: FlymFork/src/main/res/values-de/strings.xml (hunk 1 above)
Comment (SIRSteiner) [reply FlymFork/src/main/res/values-de/strings.xml]: I do not want. This happens at Oneskyapp. You can add it again.

File: FlymFork/src/main/res/values-de/strings.xml (hunk 2 above)
Comment (SIRSteiner) [reply FlymFork/src/main/res/values-de/strings.xml]: Okay, I will do it. I wonder why the english strings.xml says "Flym Fork"?

Comment (SIRSteiner) [reply FlymFork/src/main/res/values-de/strings.xml]: Done...

File: FlymFork/src/main/res/values-de/strings.xml (hunk 1 above)
Comment (SIRSteiner) [reply FlymFork/src/main/res/values-de/strings.xml]: I added it...

**Comment by SIRSteiner** [comment]
Now ready for merge?

**Comment by yanus171** [comment]
I will see English strings
С уважением, Алексей

SIRSteiner <notifications@github.com> 22 декабря 2018 г. 11:34:42 ПП написал:

**Pull request merged**